qa/pull-tester/rpc-tests.py

```diff
@@ -37,7 +38,7 @@
     # terminal via ANSI escape sequences:
     BOLD = ('\033[0m', '\033[1m')
 
-RPC_TESTS_DIR = BUILDDIR + '/qa/rpc-tests/'
+RPC_TESTS_DIR = SRCDIR + '/qa/rpc-tests/'
```

Comment:
Correct.

Though, this should be changed. Having different caches lying around in folders that happen to be your working dir right then is misleading at best.

Anyway, this is unrelated to this pull.